src/search/components/App.js

```diff
@@ -1,29 +1,15 @@
 import React from 'react';
-
-import SearchForm from './SearchForm';
-import SearchResults from './SearchResults';
-import { gettext as _ } from 'core/utils';
-
-export default class App extends React.Component {
-
-  state = {
-    query: null,
-    results: [],
-  }
-
-  handleSearch = (query) => {
-    const results = [{title: 'Foo'}, {title: 'Bar'}, {title: 'Baz'}];
-    this.setState({ query, results });
-  }
-
-  render() {
-    const { query, results } = this.state;
-    return (
-      <div className="search-app">
-        <h1>{_('Add-on Search')}</h1>
-        <SearchForm onSearch={this.handleSearch} />
-        <SearchResults results={results} query={query} />
-      </div>
-    );
-  }
-}
+import { Provider } from 'react-redux';
+import { createStore, combineReducers } from 'redux';
+import CurrentSearchPage from '../containers/CurrentSearchPage';
+import search from '../reducers/search';
+import addons from 'core/reducers/addons';
+const store = createStore(combineReducers({addons, search}));
+
+const App = () => (
+  <Provider store={store}>
```

Comment:
This redux stuff normally seems to go in an index.js file but we want to split it out per app. I'm not sure if this is the best place for it but apparently it works.